freshmaker/handlers/koji/rebuild_flatpak_application_on_module_ready.py

```diff
@@ -280,15 +280,15 @@ def _missing_composes(
                 }
 
                 if not name_stream_set.isdisjoint(module_name_stream_set):
-                    missing_composes.update(
+                    updated_composes.update(
                         mapping[name_stream]
                         for name_stream in name_stream_set.difference(
                             module_name_stream_set
                         )
                     )
-                missing_composes.update(module_name_stream_version_set)
+                updated_composes.update(module_name_stream_version_set)
 
-        return missing_composes
+        return " ".join(updated_composes)
```

Comment:
As a generic function interface, it would be more useful to return the list and let caller handle the concatenation if needed.

```suggestion
        return updated_composes
```

Reply:
Ah, I see this is meant to return source parameter for ODCS. In that case returning the string makes sense but it would be better to mention ODCS in docstring and function name.

Reply:
Yeah. I will update the docstring and function name.